tests/ha/control_pod/test_control_pod_softfailure.py

```diff
@@ -279,3 +280,166 @@ def test_45498(self):
 
         LOGGER.info("ENDED: Verify IAM user creation/deletion and IOs on already created IAM user "
                     "while performing control pod soft-failure in loop 1 by 1 for N control pods.")
+
+    @pytest.mark.ha
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-45501")
+    def test_45501(self):
```

Comment:
why test number as name?

Reply:
> why test number as name?

It was getting too long names for test for these tests. @apurwamohite Should I update to yours with test name only to keep it consistant.